tests/ffi/common/prepare.sh

```diff
@@ -37,6 +37,18 @@ reload_config() {
 }
 
 prepare_images() {
+   # Update container image_copy_tmp_dir if the image is an OStree.
+   # Default location for storing temporary container image content. Can be overridden with the TMPDIR environment
+   # variable. If you specify "storage", then the location of the
+   # container/storage tmp directory will be used.
+   # By default image_copy_tmp_dir="/var/tmp"
+   # This is a work around and it should not be used constantly
+   if [ -d /run/ostree ]; then
```

Comment:
The patch looks good to me but thinking ahead in time for saving time for us, I would like to recommend a few things:

1) A function to see if we are running in ostree or not.

```
OSTREE_PATH="/run/ostree" # global definition
 is_ostree() {
    if [ -d "${OSTREE_PATH}" ]; then
        echo "The system is running under OSTree."
        return 0
    else
        echo "The system is not running under OSTree."
        return 1
    fi
}       
```

2) Can you define global constants for:
      /var/tmp.dir
      /etc/qm/containers/containers.conf.d
      qm_image_tmp_dir.conf

Other than that, the patch looks clean and good, thanks so much for addressing this nasty bug.

Reply:
Added code as suggested.
Please review if needed.